packages/wonder-blocks-icon-button/src/components/__docs__/icon-button.stories.js

```diff
@@ -0,0 +1,201 @@
+                onClick={(e) => console.log("Click!")}
+            />
+        </View>
+    );
+};
+
+Variants.parameters = {
+    docs: {
+        storyDescription: `In this example, we have primary, secondary,
+            tertiary, and disabled \`IconButton\`s from left to right.`,
+    },
+};
+
+export const Light: StoryComponentType = () => {
+    return (
+        <View style={styles.dark}>
+            <IconButton
+                icon={icons.search}
+                aria-label="search"
+                light={true}
+                onClick={(e) => console.log("Click!")}
+            />
+        </View>
+    );
+};
+
+Light.parameters = {
+    docs: {
+        storyDescription: `An IconButton on a dark background.
+            Only the primary kind is allowed to have the \`light\`
+            prop set to true.`,
+    },
+};
+
+export const DisabledLight: StoryComponentType = () => {
+    return (
+        <View style={styles.dark}>
+            <IconButton
+                disabled={true}
+                icon={icons.search}
+                aria-label="search"
+                light={true}
+                onClick={(e) => console.log("Click!")}
+            />
+        </View>
+    );
+};
+
+DisabledLight.parameters = {
+    docs: {
+        storyDescription:
+            "This is a disabled icon button with the `light` prop set to true.",
+    },
+};
+
+export const UsingHref: StoryComponentType = () => {
+    return (
+        <IconButton
+            icon={icons.info}
+            aria-label="More information"
+            href="/"
+            target="_blank"
+            onClick={(e) => console.log("Click!")}
+        />
+    );
+};
+
+UsingHref.parameters = {
+    docs: {
+        storyDescription: `This example has an \`href\` prop in addition to the
+            \`onClick\` prop. \`href\` takes a URL or path, and clicking the
+            icon button will result in a navigation to the specified page.
+            Note that \`onClick\` is not required if \`href\` is defined.
+            The \`target="_blank"\` prop will cause the href page to open in
+            a new tab.`,
+    },
+};
+
+export const WithAriaLabel: StoryComponentType = () => {
+    return (
+        <View style={styles.arrowsWrapper}>
+            <IconButton
+                icon={icons.caretLeft}
+                onClick={(e) => console.log("Click!")}
+                aria-label="Previous page"
+            />
+            <IconButton
+                icon={icons.caretRight}
+                onClick={(e) => console.log("Click!")}
+                aria-label="Next page"
+            />
+        </View>
+    );
+};
+
+WithAriaLabel.parameters = {
+    docs: {
+        storyDescription: `By default, the icon buttons do not have
+            accessible names. The \`aria-label\` prop must be used to explain
+            the function of the button. Remember to keep the description
+            concise but understandable.`,
+    },
```

Comment:
More information will be added about this when we do the dedicated accessibility section.